tox.ini

```
@@ -1,9 +1,10 @@
 [tox]
-envlist = py38-{lilac,maple,nutmeg,olive,palm,quince},flake8,report
+envlist = py{38,311}-{lilac,maple,nutmeg,olive,palm,quince,redwood},flake8,report
```

... and we could possibly reduce the matrix here, too.

```suggestion
envlist = py38-{lilac,maple,nutmeg,olive,palm,quince},py311-redwood,flake8,report
```